src/UsblParser.cpp

```diff
@@ -410,8 +410,6 @@ DeviceSettings UsblParser::parseCurrentSettings (string const &buffer)
             settings.carrierWaveformId = atoi(splitted2.at(1).c_str());
         else if(splitted2.at(0).find("Local Address") != string::npos)
```

Comment:
Not related to this commit but .... shouldn't all these tests be

```
  if (.....find() == known_offset)
```

or even

```
splitted2.at(0) == "Cluster Size"
```

You known where the string should be in the message ... don't you ?